frontend/src/components/layout/Header/Header.styled.ts

```diff
@@ -65,16 +112,19 @@ export const HeroHeading = styled(GradientHeading)`
   max-width: 18rem;
 
   @media (min-width: ${breakpoints.md}) {
-    font-size: 3.5rem;
+    font-size: 3.2rem;
     line-height: 1;
-    padding-right: 0.5rem;
+    padding-right: 3rem;
     max-width: 34rem;
     background-size: 100%;
   }
 
   @media (min-width: ${breakpoints.lg}) {
-    font-size: 4.3rem;
+    font-size: 3.2rem;
     line-height: 1;
-    max-width: 42rem;
+    text-align: left;
+    max-width: 42.5rem;
```

Comment:
Same here, I think max-width make things more difficult to maintain

Reply:
Same as above.